tests/system/action/committee/test_json_upload.py

```diff
@@ -0,0 +1,548 @@
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {
+                "name": "test",
+                "meeting_name": "test meeting",
+                "meeting_template": {
+                    "value": "test",
+                    "info": ImportState.DONE,
+                    "id": 23,
+                },
+            },
+        }
+        assert response.json["results"][0][0]["statistics"][5] == {
+            "name": "Meetings copied from template",
+            "value": 1,
+        }
+        assert response.json["results"][0][0]["statistics"][4] == {
+            "name": "Meetings created without template",
+            "value": 0,
+        }
+
+    def test_json_upload_committee_managers(self) -> None:
+        self.set_models({"user/23": {"username": "test"}})
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "committee A",
+                        "committee_managers": "test, new",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": ["Missing committee manager(s): new"],
+            "data": {
+                "name": "committee A",
+                "committee_managers": [
+                    {"value": "test", "info": ImportState.DONE, "id": 23},
+                    {"value": "new", "info": ImportState.WARNING},
+                ],
+            },
+        }
+        assert response.json["results"][0][0]["statistics"][6] == {
+            "name": "Committee managers relations",
+            "value": 1,
+        }
+
+    def test_json_upload_organization_tags(self) -> None:
+        self.set_models({"organization_tag/37": {"name": "test"}})
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "committee A",
+                        "organization_tags": "test, new",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {
+                "name": "committee A",
+                "organization_tags": [
+                    {"value": "test", "info": ImportState.DONE, "id": 37},
+                    {"value": "new", "info": ImportState.NEW},
+                ],
+            },
+        }
+        assert response.json["results"][0][0]["statistics"][0] == {
+            "name": "Tags created",
+            "value": 1,
+        }
+
+    def test_json_upload_forward_to_committees(self) -> None:
+        self.set_models({"committee/37": {"name": "test"}})
```

Comment:
Example for add, remove
```
self.set_models({
    "committee/36": {"name": "committee A", "forward_to_committee_ids": [38,], },
    "committee/37": {"name": "test"},
    "committee/38": {"name": "should be removed", "receive_forwardings_from_committee_ids": [36]},
}
```
We should show that the request will remove the existing forwarding from "committee A" to committee/38 "should be removed". As new line in forward_to_committee {"value": "should be removed", "info": ImportState.REMOVE, "id": 38} or as message in messages

Reply:
I have updated the test and include the REMOVE entry in the forward_to_committees.